BE/src/main/java/louie/dong/airbnb/book/Book.java

```diff
@@ -22,4 +27,19 @@ public class Book {
 	private LocalDateTime checkIn;
 	private LocalDateTime checkOut;
 	private int guestCount;
+	private int finalPrice;
+	private boolean isCanceled;
+
+	public Book(Accommodation accommodation, LocalDateTime checkIn, LocalDateTime checkOut,
+		int guestCount, int finalPrice) {
+		this.accommodation = accommodation;
+		this.checkIn = checkIn;
+		this.checkOut = checkOut;
+		this.guestCount = guestCount;
+		this.finalPrice = finalPrice;
+	}
+
+	public void changeCanceled(boolean isCanceled) {
+		this.isCanceled = isCanceled;
+	}
```

Comment:
setter 느낌의 메소드보다는 한가지 동작을 하는 메소드로 변경 해보는건 어떨까요?
(위 코멘트에서 언급한 파라미터 0개인 메소드)
```suggestion
	public void cancel() {
		this.isCanceled = true;
	}
```

Reply:
한가지 동작을 하는 메소드를 사용한다면 코드를 알아보기 쉬울 것 같네요!
왕민 말씀대로 수정해보겠습니다 🤩